test/fixtures/examples/jsx/actual.js

```diff
@@ -2,7 +2,7 @@ otherCondition() -> true
 
 ThingsList1({ things, selectedId }) =>
   <ul>
-    {[for thing of things:
+    {[for const thing of things:
```

Comment:
again, would prefer idiomatic `elem thing in things` especially in the `examples` folder but I can clean these up later